src/app/pages/component-sidenav/component-sidenav.html

```diff
@@ -1,9 +1,9 @@
 <md-sidenav-container class="docs-component-viewer-sidenav-container">
-  <md-sidenav #sidenav class="docs-component-viewer-sidenav"
+  <md-sidenav aria-label="Side navigation" role="navigation" #sidenav class="docs-component-viewer-sidenav"
```

Comment:
I don't think that this is necessarily a problem. To my knowledge, a `<nav>` element or an element with `role="navigation"` just cannot have a `<main>` or `role="main"` as a descendant. Adding the role navigation here to allow screen readers to see the whole side nav, and then dig down into the individual sections of the side nav was suggested based off this issue: https://github.com/angular/material.angular.io/issues/111